4-perftools/Dockerfile.rhel7

```diff
@@ -38,7 +37,7 @@ USER 0
 
 WORKDIR ${HOME}
 
-COPY contrib/etc/scl_enable /root/etc/scl_enable
+COPY root/root/etc/scl_enable /root/etc/scl_enable
```

Comment:
This is not needed IMPOV - `scl_enable` is already in `/root/etc/scl_enable` by `COPY ./root/ /`